### PR TITLE
revise doc

### PR DESCRIPTION
**Pull request by JimmyCYJ**
In the Ingress SDS user guide, the first section introduces how to create a k8s secret to provide key/cert for httpbin gateway, and then replace the key/cert by deleting and creating a new k8s secret. The second section assumes the ingress gateway is using the original key/cert, not the new one. 

Add a missing step which restores the original key/cert for httpbin at the beginning of the second section.

### CODE REVIEW

File: content/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md [hunk 1]
```diff
@@ -302,6 +302,15 @@ You can configure an ingress gateway for multiple hosts,
 `httpbin.example.com` and `helloworld-v1.example.com`, for example. The ingress gateway
 retrieves unique credentials corresponding to a specific `credentialName`.
 
+1.  Restore credentials for httpbin by deleting its secret and creating again.
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md]:
```suggestion
1.  To restore the credentials for `httpbin`, delete its secret and create it again.
```

**Comment by JimmyCYJ** [comment]
Thanks for the review! Please take a look.

**Comment by JimmyCYJ** [comment]
https://github.com/istio/istio/issues/13439

**Comment by liminw** [comment]
https://github.com/istio/istio.io/issues/4404

**Pull request merged**